software/base/src/main/java/org/apache/brooklyn/entity/software/base/VanillaWindowsProcess.java

```diff
@@ -103,7 +103,16 @@
     // TODO If automatic updates are enabled and there are updates waiting to be installed, thirty minutes may not be sufficient...
     ConfigKey<Duration> REBOOT_COMPLETED_TIMEOUT = ConfigKeys.newDurationConfigKey("reboot.completed.timeout",
             "duration to wait whilst waiting for a machine to finish rebooting, and thus to become available again", Duration.minutes(30));
-    
-    AttributeSensor<Integer> RDP_PORT = Sensors.newIntegerSensor("rdpPort");
-    AttributeSensor<Integer> WINRM_PORT = Sensors.newIntegerSensor(WinRmTool.PROP_PORT.getName());
+
+    AttributeSensor<Integer> RDP_PORT = Sensors.newIntegerSensor("rdp.port");
+    AttributeSensor<Integer> WINRM_PORT = Sensors.newIntegerSensor("winrm.port");
+
+    @Deprecated
```

Comment:
Please include javadoc saying why and when it was deprecated, and what to use instead. Also include a description for the sensor saying it's deprecated. Same for the others below. For example:

    /**
     * @deprecated since 0.9.0; use {@link #RDP_PORT} instead.
     */
    @Deprecated
    AttributeSensor<Integer> RDP_PORT_CAMEL_CASE = Sensors.newIntegerSensor(
            "rdpPort",
            "[DEPRECATED] instead use 'rdp.port'");